_posts/2016-11-01-security-alert-solidity-variables-can-overwritten-storage.md

```diff
@@ -4,8 +4,6 @@ title: 'Security Alert: Variables can be overwritten in storage'
 date: 2016-11-01T08:40:44+00:00
 author: Christian Reitwiessner
 layout: post
-guid: https://blog.ethereum.org/?p=3363
-permalink: /2016/11/01/security-alert-solidity-variables-can-overwritten-storage/
```

Comment:
I don't think there's any reason to keep the old URL (the reason for that was seemingly at the time of original posting a typo stayed in it and nobody wanted to break the URL).

This blog is a new blog.

Reply:
oh no actually most of that stuff, especially harmonizing the headers, I have done already in this PR which is still waiting (because of the category stuff) https://github.com/ethereum/solidity-blog/pull/44

Reply:
I see these two lines being the only conflicting ones.

Reply:
ok perfect, yeah I only worked on the headers!

Reply:
and added the "originally published at" links in the beginning of each old post